content/config/otel.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Open Telemetry for the Docker CLI
+description: Learn about how you can leverage the Docker's capabilities to capture OTel traces
+keywords: otel, telemetry, traces, metrics, logs
+---
+
+{{< introduced engine 26.1.0 >}}
+
+The Docker CLI supports [OTel](https://opentelemetry.io/docs/) instrumentation
+for emitting metrics about command invocations. This is disabled by default.
+You can configure the CLI to start emitting metrics to the endpoint that you
+specify. This allows you to capture information about your `docker` command
+invocations for more insight into your Docker usage.
+
+Exporting metrics is opt-in, and you control where data is being sent by
+specifying the destination address of the metrics collector.
+
+## What is OTel?
+
+OTel stands for Open Telemetry. OTel is an open observability framework for
```

Comment:
OpenTelemetry should be one word. FWIW I also think that it's probably better to canonically refer to it as `OpenTelemetry` throughout the docs, perhaps only showing `(OTel)` as a shortened version somewhere.

While it's definitely referred to that informally, and they themselves use the term (e.g. OTel Dev Days is highlighted on their site right now), it's kind of like referring to "Prometheus" as "Prom" or "Kubernetes" as "K8s", so I think using the full OpenTelemetry project name is proper even if a bit of a mouthful 🙃

Reply:
Makes sense, thanks for the review @milas - always appreciated